### ISSUE TITLE
validation loss

### ISSUE THREAD

**Issue by soans1994** [brianhuang1019/FCN-pytorch#28]
hello author,

can you please tell me your validation loss or iou. I implemented original fcn model and also tried yours. The training scores are good. But the validation iou score is less 0.1. I tried shuffling data and everything. Why this happens?
please please give me your advice.

Thank you
Soans
